### PR TITLE
Cavonac/offmig

### PR DESCRIPTION
**Pull request by cavonac**
updated offline migration notebook and associated documentation

### CODE REVIEW

File: extensions/azurehybridtoolkit/notebooks/hybridbook/content/appendices.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 See the <a href="https://azure.microsoft.com/en-us/global-infrastructure/locations/">Azure locations</a> page for a complete list of Azure regions along with their general physical location. The following is a list of common North American location settings for this guide:
 
 ### Regions
-| Setting | Location |
+| Setting | Location |
```
Comment (smartguest) [inline extensions/azurehybridtoolkit/notebooks/hybridbook/content/appendices.md:8]: Remove the space here using notepad.

**Review by smartguest — APPROVED** [review]

File: extensions/azurehybridtoolkit/notebooks/hybridbook/content/appendices.md (hunk 1 above)
Comment (cavonac) [reply extensions/azurehybridtoolkit/notebooks/hybridbook/content/appendices.md]: removing file. the changes were migrated to a notebook in another PR

**Review by kburtram — APPROVED** [review]

**Comment by kburtram** [comment]
@cavonac do you have permissions to merge yourself?  or do you need someone to merge for you?

**Comment by smartguest** [comment]
@kburtram I'll merge it for @cavonac

**Pull request merged**